### PR TITLE
#6 HtAutoRedirect introduced

### PR DESCRIPTION
**Pull request by Vatavuk**
Introduced `HtAutoRedirect` for #6

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#26](https://github.com/yegor256/cactoos-http/issues/26) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#26](https://github.com/yegor256/cactoos-http/issues/26) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol), here is [why](https://www.0crat.com/footprint/C63314D6Z/952139108); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

**Comment by Vatavuk** [comment]
@neonailol ping

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 6days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
The user @neonailol[/z](https://www.0crat.com/u/neonailol) resigned from [#26](https://github.com/yegor256/cactoos-http/issues/26), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -15 point(s) just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
This pull request [#26](https://github.com/yegor256/cactoos-http/issues/26) is assigned to @sharpler[/z](https://www.0crat.com/u/sharpler), here is [why](https://www.0crat.com/footprint/C63314D6Z/1101543830); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

**Comment by Vatavuk** [comment]
@SharplEr ping

**Comment by 0crat** [comment]
@sharpler[/z](https://www.0crat.com/u/sharpler) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

File: src/test/java/org/cactoos/http/HtAutoRedirectTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import org.takes.tk.TkText;
+
+/**
+ * Test case for {@link HtAutoRedirect}.
+ *
+ * @author Vedran Vatavuk (123vgv@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```
Comment (SharplEr) [inline src/test/java/org/cactoos/http/HtAutoRedirectTest.java]: @Vatavuk  You can remove this line without problems

**Comment by SharplEr** [comment]
@Vatavuk  Please take a look at my remarks.

**Comment by Vatavuk** [comment]
@SharplEr fixed in 768cbde

**Comment by SharplEr** [comment]
@llorllale  good to merge
@Vatavuk  Thanks.

**Comment by SharplEr** [comment]
@llorllale ping

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @sharpler[/z](https://www.0crat.com/u/sharpler), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#26](https://github.com/yegor256/cactoos-http/issues/26) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by elenavolokhova** [comment]
@SharplEr According to our Policy:
>The code reviewer found at least three problems in the code

Please, confirm that you'll try to find 3 issues next time.

**Comment by elenavolokhova** [comment]
@Vatavuk According to our Policy:

>All commit messages have links to issues

Please, confirm that you'll include link to issue in all commits in future, even in fixes of issues found by reviewer.

**Comment by Vatavuk** [comment]
@elenavolokhova I confirm

**Comment by SharplEr** [comment]
@elenavolokhova confirm

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @sharpler[/z](https://www.0crat.com/u/sharpler)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**